cms/templates/studio_xblock_wrapper.html

```diff
@@ -62,6 +67,12 @@
                                 <span class="action-button-text">${_("Edit")}</span>
                             </a>
                         </li>
+                        <li class="action-item action-visibility">
+                            <a href="#" data-tooltip="${_("Visibility Settings")}" class="visibility-button action-button">
```

Comment:
I visual class gets added here when Visibility settings have been customized.  However, there is no text equivalent.  Could we add screen reader text that appends "Modified" or "Customized" to the word "Visibility" below or something to indicate that visibility has been modified?